src/keyring/KeyringController.test.ts

```diff
@@ -449,8 +563,42 @@ describe('KeyringController', () => {
     } catch (e) {
       error2 = e;
     }
-    expect(error1.message).toContain('Keyring Controller signTypedMessage:');
-    expect(error2.message).toContain('Keyring Controller signTypedMessage:');
+
+    expect((error1 as ErrorMessageObject).message).toContain(
+      'Keyring Controller signTypedMessage:',
+    );
+
+    expect((error2 as ErrorMessageObject).message).toContain(
+      'Keyring Controller signTypedMessage:',
+    );
+  });
+
+  it('should fail in signing message when from address is not provided', async () => {
+    let error: unknown = { message: '' };
+    try {
+      const typedMsgParams = [
+        {
+          name: 'Message',
+          type: 'string',
+          value: 'Hi, Alice!',
+        },
+        {
+          name: 'A number',
+          type: 'uint32',
+          value: '1337',
+        },
+      ];
+      await keyringController.signTypedMessage(
+        { data: typedMsgParams, from: '' },
+        SignTypedDataVersion.V1,
+      );
+    } catch (e) {
+      error = e;
+    }
+
+    expect((error as ErrorMessageObject).message).toContain(
```

Comment:
Same for this test — use `expect(...).rejects.toThrow(...)`?

Reply:
Fixed